static_topology/stackstate_checks/static_topology/static_topology.py

```diff
@@ -0,0 +1,136 @@
+# (C) StackState 2020
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import csv
+import codecs
+
+from stackstate_checks.base import ConfigurationError, AgentCheck, TopologyInstance
+
+
+class StaticTopologyCheck(AgentCheck):
+    SERVICE_CHECK_NAME = "StaticTopology"
+    INSTANCE_TYPE = "StaticTopology"
+
+    def get_instance_key(self, instance):
+        if "components_file" not in instance:
+            raise ConfigurationError('Static topology instance missing "components_file" value.')
+
+        return TopologyInstance(self.INSTANCE_TYPE, instance["components_file"])
+
+    def check(self, instance):
+        if 'components_file' not in instance:
+            raise ConfigurationError('Static topology instance missing "components_file" value.')
```

Comment:
we don't throw exception from the check but rather use the `service_check` critical

Reply:
@TizianoPerrucci  this is the config param check and we always raise `ConfigurationError`.